code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1292,10 +1294,8 @@
 				if((E.type == OE.type)) // Type has to be explicit, as right limbs are a child of left ones etc.
 					var/brute = part[2]
 					var/burn = part[3]
-					var/IB = part[4]
 					//Deal the damage to the new organ and then delete the entry to prevent duplicate checks
 					E.receive_damage(brute, burn, ignore_resists = TRUE)
-					E.internal_bleeding = IB
```

Comment:
This sorta looks like it was removed on accident

Reply:
was an accident *however* this code is bugged. no damage is carried over between species swaps.